data/campaigns/Heir_To_The_Throne/scenarios/17_Scepter_of_Fire.cfg

```diff
@@ -617,6 +617,71 @@
         {CLEAR_VARIABLE delfador_moves}
     [/event]
 
+    # Introductory dialog to give the player a hint that she may lose units to
+    # {EXPAND_LAVA}.
+    [event]
+        name=turn 2
+        # The noise was played by {EXPAND_LAVA}.
+        [message]
+            id=Konrad
+            {KONRAD_VARIATION concerned}
+            message=_ "That noise was made by no troll! What manner of beast makes its den in these forsaken caves?"
+        [/message]
+        [message]
+            # Comic relief
```

Comment:
Maybe `# po: Comic relief`, so that the hint is given to the translators when they're working in poedit?

Thanks for your tolerance with all these suggestions for changes.